src/bin/server.cpp

```diff
@@ -1,36 +1,41 @@
-#include <boost/asio/io_service.hpp>
-
-#include <cstdio>
-#include <cstdlib>
-#include <iostream>
+#include "cxxopts.hpp"
 
 #include "hyrise.hpp"
 #include "scheduler/node_queue_scheduler.hpp"
 #include "server/server.hpp"
-#include "utils/load_table.hpp"
+
+cxxopts::Options get_server_cli_options() {
+  cxxopts::Options cli_options("./hyriseServer", "Starts Hyrise Server in order to accept network requests.");
+
+  // clang-format off
+  cli_options.add_options()
+    ("help", "Display this help and exit")
+    ("p,port", "Specify the port number. 0 means randomly select an available one", cxxopts::value<uint16_t>()->default_value("5432"))  //NOLINT
```

Comment:
Good point. However, setting the port to 0 the makes the OS selecting a random port (not the application). The range can be set in `/proc/sys/net/ipv4/ip_local_port_range` (default: 40000 - 60999 [Ubuntu 19.04]). I'd like to continue using this OS functionality in order to avoid bloating the code.